docs/platforms/javascript/guides/react/features/react-router/v6.mdx

```diff
@@ -19,18 +19,21 @@ Update your `Sentry.browserTracingIntegration` to `Sentry.reactRouterV6BrowserTr
 To ensure proper routing instrumentation, initialize Sentry by calling `Sentry.init` **before**:
 - Wrapping your `<Routes />` component
 - Using `useRoutes`
-- Using `createBrowserRouterV6`
+- Using `wrapCreateBrowserRouterV6` or `wrapCreateMemoryRouterV6`
 
 </Alert>
 
-### Usage with `createBrowserRouter`
+### Usage with `createBrowserRouter` or `createMemoryRouter`
 
-<Alert level="info">
-Available in `@sentry/react` version `7.21.0` and above.
-</Alert>
+If you choose to create your router instance with [`createBrowserRouter`](https://reactrouter.com/en/main/routers/create-browser-router) or [`createMemoryRouter`](https://reactrouter.com/en/main/routers/create-memory-router), you can use `Sentry.wrapCreateBrowserRouterV6` or `Sentry.wrapCreateMemoryRouterV6` to wrap it with the instrumentation:
+
+<Alert level="warning" title="Note">
+
+`wrapCreateMemoryRouterV6` is available from `@sentry/react` version `8.49.0`. For versions prior to `8.49.0`, you can use `wrapCreateBrowserRouterV6` with `createMemoryRouter`.
```

Comment:
> you can use `wrapCreateBrowserRouterV6` with `createMemoryRouter`.

is that true actually?

Reply:
Yes, we were recommending that prior to this update. (Without the support of `initialEntries` though)

Reply:
Ah right, for some reason I thought it was completely broken. I would maybe rephrase this so it is more specific as to what to do:

```suggestion
`wrapCreateMemoryRouterV6` was introduced in SDK version 8.49.0. Prior to that version, we suggested using `wrapCreateBrowserRouterV6` with `createMemoryRouter`. If you are currently using `wrapCreateBrowserRouterV6` to wrap `createMemoryRouter`, it is recommended that you use `wrapCreateMemoryRouterV6` instead.
```

Reads horribly though 😂